recipes/earth-osm/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "earth-osm" %}
+{% set version = "0.0.8" %}
+{% set sha = "b0fbd294fa4ad02df07510b0e1494fd665ffd6aa6adc7712e9dcf08844881b75" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/earth_osm-{{ version }}.tar.gz
+  sha256: {{ sha }}
```

No need for a jinja variable that it used only once.

```suggestion
  sha256: b0fbd294fa4ad02df07510b0e1494fd665ffd6aa6adc7712e9dcf08844881b75
```